core/modules/field/modules/text/text.module

```diff
@@ -411,6 +411,10 @@ function text_summary($text, $format_id = NULL, $size = 600) {
   }
   $break_points[] = $line_breaks;
 
+  // Typing a number after the ؟ character (Arabic question mark) causes text to
+  // be reversed mid-line. That's the reason why the last element in this array
+  // may seem odd. It actually produces valid code.
+  // See https://github.com/backdrop/backdrop-issues/issues/4546
   // If the first paragraph is too long, split at the end of a sentence.
```

Comment:
I think this comment line should appear first, then your addition.

Reply:
Done 👍